Formula/mgis.rb

```diff
@@ -0,0 +1,60 @@
+class Mgis < Formula
+  desc "Provide tools to handle MFront generic interface behaviours"
+  homepage "https://thelfer.github.io/mgis/web/index.html"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/MFrontGenericInterfaceSupport.git", using: :git, branch: "master"
+
+  stable do
+    url "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-2.0.tar.gz"
+    sha256 "cb427d77f2c79423e969815b948a8b44da33a4370d1760e8c1e22a569f3585e2"
+    patch :DATA
```

Comment:
Is there an upstream version of this?

Reply:
looks like it is [this commit](https://github.com/thelfer/MFrontGenericInterfaceSupport/commit/3861ac90c76d69956b7267a8d0cc663bdf2e3689), but the commit has some syntax issue, not being released yet.

Reply:
@SMillerDev, yes [here](https://github.com/thelfer/MFrontGenericInterfaceSupport/commit/bfeb0ab537ef2870914986d193854fe0d810d5d2)

Reply:
Could we use that version then?

Reply:
@SMillerDev I'm asking when it will be released

Reply:
@SMillerDev no answer for the moment but the patch has been approved upstream